.github/workflows/deploy.yml

```diff
@@ -188,5 +188,5 @@ jobs:
 
       - name: Push Images
         run: |
-          docker login --username=${{ secrets.DOCKER_USER }} --password=${{ secrets.DOCKER_TOKEN }}
+          docker login --username=anatolkaralkou --password=${{ secrets.DOCKERHUB_BRIGHT_TOKEN }}
```

Comment:
Let's introduce a pair of 2 env:
- `secrets.DOCKER_BRIGHT_USER` 
- `secrets.DOCKER_BRIGHT_TOKEN`
to align names with old ones.

Reply:
Let's keep it as is. Because username isn't a secret, and it'll be hard to determine which username is used for working with Docker Hub